committee-code-of-conduct/governance/onboarding-offboarding.md

```diff
@@ -0,0 +1,43 @@
+# Onboarding and offboarding new Code of Conduct Committee members
+
+Different actions on this list must be carried out by different members: 
+
+- **Outgoing members:** members who are ending their term
+- **Incoming members:** members beginning their term
+- **Carryover members:** members mid-way through their term
+
+## Permissions 
+
+### Slack
+
+**Who executes:** When offboarding, outgoing members must remove themselves from Slack channels. When onboarding, carryover members must add incoming members.
+
+- [ ] Code of conduct committee Slack channel on `kubernetes.slack.com`
+- [ ] Code of conduct sync Slack channel on `cloud-native.slack.com`
+
+### Kubernetes/community permissions and google permissions
+
+**Who executes:** Carryover members should ensure that outgoing members are removed and incoming members are invited. 
+
+- [ ] Update `kubernetes/community` [`sigs.yaml`](https://github.com/kubernetes/community/blob/master/sigs.yaml) 
+- [ ] Update `kubernetes/k8s.io` [`groups.yaml`](https://github.com/kubernetes/k8s.io/blob/master/groups/groups.yaml)
+
+    > **Note:** This file controls access to the mailing list and Google Drive! If you do not update this file, adding people manually (via the UIs) continually revert until this file is updated.
```

Comment:
Should note using https://cs.k8s.io/ to search out additional places.  I updated OWNERS_ALIASES in k/k8s.io, k/org, and k/test-infra.

Reply:
Suggest changing "Google Drive" to "Google folder share" or whatever is the right Google Noun for that.  We don't have a Google Drive.